src/context_diff.rs

```diff
@@ -719,6 +741,7 @@ mod tests {
         }
     }
 
+    #[ignore]
```

Comment:
why ignore ?

Reply:
Since, now we also have modification time in the `brief`. We can't really compare it with a constant value

Reply:
the test should be updated to use regex and to ignore dates

Reply:
Okay, will do! Thanks :)